sdk/openai/openai-assistants/test/public/openaiassistants.spec.ts

```diff
@@ -0,0 +1,432 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Recorder } from "@azure-tools/test-recorder";
+import { assert, matrix } from "@azure/test-utils";
+import { assertAssistantEquality } from "./utils/asserts.js";
+import { AuthMethod, createClient, startRecorder } from "./utils/recordedClient.js";
+import { getModels } from "./utils/utils.js";
+import { Context } from "mocha";
+import { AssistantsClient } from "../../src/assistantsClient.js";
+
+describe("OpenAIAssistants", () => {
+  let recorder: Recorder;
+  let models: string[] = [];
+
+  beforeEach(async function (this: Context) {
+    recorder = await startRecorder(this.currentTest);
+    if (!models.length) {
+      models = await getModels(recorder);
+    }
+  });
+
+  afterEach(async function () {
+    await recorder.stop();
+  });
+
+  // const authTypes = ["AzureAPIKey", "OpenAIKey", "AAD"] as AuthMethod[];
+  const authTypes = ["OpenAIKey" as AuthMethod];
+
+  matrix([authTypes] as const, async function (authMethod: AuthMethod) {
```

Comment:
nitpick: lifting the const should be enough I think:
```suggestion
  const authTypes = ["OpenAIKey"] as const;

  matrix([authTypes], async function (authMethod: AuthMethod) {
```